src/screen/tips/TipDetailScreen.tsx

```diff
@@ -128,7 +128,7 @@ function TipDetailScreen({route}: ScreenProps) {
         data={tip.tips}
         style={[globalStyles.paddedContainer, styles.container]}
         ItemSeparatorComponent={Divider}
-        renderItem={({item}) => {
+        renderItem={({item}: {item: any[]}) => {
```

Comment:
Let's use `FlatList` instead to get the correct types

Reply:
done ✅